cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-media.php

```diff
@@ -1312,15 +1369,130 @@ public function delete_post_meta( $post_id, $key ) {
 		delete_post_meta( $post_id, $key );
 	}
 
+	/**
+	 * Get the breakpoint generation options for an attachment.
+	 *
+	 * @param int $attachment_id The attachment ID.
+	 *
+	 * @return array
+	 */
+	public function get_breakpoint_options( $attachment_id ) {
+		// Add breakpoints if we have an image.
+		$breakpoints     = array();
+		$has_breakpoints = $this->plugin->config['settings']['global_transformations']['enable_breakpoints'];
+
+		if ( 'off' !== $has_breakpoints && wp_attachment_is_image( $attachment_id ) ) {
```

Comment:
You can turn this line into:
```suggestion
		if ( $has_breakpoints && 'off' !== $has_breakpoints && wp_attachment_is_image( $attachment_id ) ) {
```
And then remove the `if` on line 1395.

Reply:
Actually, this was partly pulled from the old code and I didn't check it fully. line 1395 is already not needed since it shouldn't even get here if it is off. I'll revise this. Thanks for the catch!